src/__tests__/network.integration.ts

```diff
@@ -0,0 +1,69 @@
+import { execSync } from "child_process"
+import { Message } from "../message"
+import { CborData } from "../message/cbor"
+import { Base, Network } from "../network"
+import { ID1, ID_RND, SERVERS } from "./data"
+
+describe("Network", () => {
+  let network: Network
+  let HEX: string
+  let CBOR_DATA: CborData
+
+  beforeAll(() => {
+    const hex = execSync("../many-rs/target/debug/many message status --hex")
+    HEX = `d2${hex.toString()}`
+    CBOR_DATA = Buffer.from(HEX, "hex")
```

Comment:
This assumes `many-js` is adjacent to `many-rs` in your file structure, which might not be true.

Reply:
You don't need to rely on `many-rs` for this.

The request will look like this

8440a053d92711a2036673746174757305c1**1a642b22b6**40

where the bytes in **bold** represent the timestamp. This is the only part of that request that changes.

All you need to do is update those bytes with the current cbor-encoded epoch timestamp.